3 - PB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1665,6 +1665,59 @@ Gli argomenti principali trattati nel documento sono due:
 
 Nel documento dovranno essere descritti nel dettaglio i design pattern utilizzati nel prodotto e derivati dalle tecnologie, inserendo anche i relativi diagrammi UML, e ogni altro aspetto progettuale che sia rilevante ai fini dell'architettura e del design del prodotto.
 
+
+== Processo di verifica <processo_verifica>
+
+_Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.4.9_
+
+=== Scopo
+Il processo di verifica ha lo scopo di dimostrare la correttezza e l'adempimento di tutti i requisiti di un elemento di sistema, definiti in #adr_v, rispettando quanto definito nel #pdq_v.
+Nel caso in cui vengano identificati errori o mancanze, vengono fornite le informazioni necessarie a determinare la risoluzione delle anomalie riscontrate.
+
+Tali operazioni sono possibili mediante molteplici test automatici eseguiti in parte durante il [@processo_integrazione].
+Così facendo si garantisce l'individuazione di eventuali problemi tempestivamente, evitandone un accumulo indesiderato.
+
+=== Risultati
+Come risultato della corretta applicazione del processo di verifica, per ogni elemento:
+- viene effettuata una verifica manuale da parte di uno o più Verificatori;
+- nel caso in cui il soggetto della verifica sia un elemento software:
+  - vengono eseguiti i test automatici di unità;
+  - vengono eseguiti i test automatici di regressione;
+  - vengono eseguiti i test automatici di integrazione;
+- vengono rilevati eventuali problemi legati al rispetto della correttezza, dei requisiti o dell'architettura, in funzione di quanto definito;
+- in caso di errori vengono riportati i dati che forniscono informazioni per le azioni correttive;
+- in caso di mancanza di anomalie viene fornita evidenza che il sistema verificato garantisce piena conformità.
+
+
+=== Attività
+==== Esecuzione
+
+La prima attività di verifica avviene tramite delle GitHub Actions eseguite automaticamente alla pubblicazione di una pull request e alla sua approvazione da parte di un Verificatore.
+Esse eseguono dei test statici sulla correttezza dell'elemento analizzato, fornendo risultati sui singoli esiti dei test.
+
+I Verificatori approvano le modifiche successivamente ad un'analisi manuale che garantisca la conformità con le norme stabilite.
+
+Successivamente vengono eseguite automaticamente GitHub Actions atte allo svolgimento di ulteriori test dipendenti dalla natura delle modifiche soggette a verifica.
+Se queste ultime comprendono elementi software, vengono eseguiti test di unità e regressione.
+Inoltre a seguito dell'integrazione del codice, vengono eseguiti test di integrazione.
+
+Le GitHub Actions designate all'esecuzione dei test sono codificate nel file _test_nodejs.yml_.
```

Comment:
la action è una, vengo descritti i job nel file; in ogni caso secondo me è meglio dire che il codice della action è descritta in quel file